### PR TITLE
ci(github-actions): support continuous deployment to multiple environments

### PR DESCRIPTION
**Pull request by Paz-X**

### CODE REVIEW

File: .github/workflows/BuildNDeployDev.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Test Ruby Installation
```
Comment (jaylenw) [inline .github/workflows/BuildNDeployDev.yml]: Let's reflect what's occurring here with a better name.

File: .github/workflows/BuildNDeployDev.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+name: Test Ruby Installation
+
+on: # run this workflow when a push has been made to `Jekyll-Alf` branch
+  push:
+    branches:
+      - Alfa-Jekyll
```
Comment (jaylenw) [inline .github/workflows/BuildNDeployDev.yml]: Change to `development` branch.

File: .github/workflows/BuildNDeployDev.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+on: # run this workflow when a push has been made to `Jekyll-Alf` branch
+  push:
+    branches:
+      - Alfa-Jekyll
+
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
```
Comment (jaylenw) [inline .github/workflows/BuildNDeployDev.yml]: set to `ubuntu-20.04`

**Review by jaylenw — COMMENTED** [review]
Make the appropriate changes mention in the comments for the production GitHub Actions yml file as well.

**Review by jaylenw — APPROVED** [review]
Approved with the latest changes.

**Review by RV-LACity — APPROVED** [review]
Looks good.

**Pull request merged**